charts/reform-scan-notification-service/values.yaml

```diff
@@ -24,7 +24,8 @@ java:
     S2S_URL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
     NOTIFICATIONS_CONSUME_TASK_DELAY_IN_MS: 3000
     NOTIFICATIONS_CONSUME_TASK_ENABLED: true
-
+    PENDING_NOTIFICATIONS_TASK_DELAY_IN_MS: 600000 # 10min
```

Comment:
are we planning to write any functionl tests for pending notifications tasks then we need to tweak values for aat  ?

Reply:
that's in flux pr ;)

Reply:
that would only apply to AAT-Prod container's. As a scheduled jobs are running(staging + prod) you would need to override it in aat template and flux.
We can set default value to what is applicable to most of the environments and only override in flux for prod envrironment.

Reply:
we can swap those - not a problem